### ISSUE TITLE
GPU environment install instructions incorrect for recent GPUs

### ISSUE THREAD

**Issue by jeffreymcallister** [intel-gpu/documentation#34]
dgpu-docs shows how to install Media SDK packages.  Media SDK is only intended for legacy GPUs.  

For example, in the instructions for Ubuntu 20.04 here: https://dgpu-docs.intel.com/installation-guides/ubuntu/ubuntu-focal.html

The libmfxgen1 package should be installed to cover Xe and newer GPUs.

**Comment by tripzero** [comment]
The legacy package in question is libmfx1?  Should be changed to libmfxgen1?  Thanks.

**Comment by jeffreymcallister** [comment]
Yes, libmfxgen1 should be installed for Xe and newer.  If you need one script for all platforms you can install libmfx1 and libmfxgen1.   Our dispatcher will figure out the right thing to load for the platform.

**Comment by eero-t** [comment]
I think this can be closed.

**Comment by bouachalazhar** [comment]
> I think this can be closed.

I have the same problem check this [link ](https://github.com/intel-gpu/documentation/issues/76#issue-1655635305) please, thanks !
